model/resource/k8s.yaml

```diff
@@ -33,6 +33,7 @@ groups:
     attributes:
       - ref: k8s.pod.uid
       - ref: k8s.pod.name
+      - ref: k8s.pod.labels
```

Comment:
Is this the correct way to add an attribute key prefix to the model? I am just wondering if the model has a different way to indicate prefixes/dynamic attr keys from constant attribute keys to help with code generation for semantic-conventions.

Reply:
@TylerHelmuth @ChrsMark @AlexanderWert can you folks please confirm? I'm new.

Reply:
`http.request.header.*` is an already existent example which follows this pattern and it is defined at https://github.com/open-telemetry/semantic-conventions/blob/04c8e49dc4fd359c1fd42e2a37c2d685ea9cfeb9/model/trace/http.yaml#L37 and https://github.com/open-telemetry/semantic-conventions/blob/04c8e49dc4fd359c1fd42e2a37c2d685ea9cfeb9/model/registry/http.yaml#L15. So I would say that's so far the way to handle dynamic attribute keys.

However, I'm not aware of any issues that this pattern could cause with tooling and code generation (as mentioned at https://github.com/open-telemetry/semantic-conventions/pull/494#issuecomment-1821581323) 🤔 but based on the fact that `http` is marked as stable this shouldn't be the case.

I would wait for more confirmations on this :).

Reply:
For the markdown tables, everything looks correct in this PR. For the code generation though, that's another thing. I don't think the tooling is handling these today in any special way so I wonder what happens. I know that @jack-berg stumbled on an unrelated bug after the HTTP stabilization release with the tool. I wonder how it is working for you on such attributes Jack?

Reply:
@joaopgrassi For code generation template attributes are a separate set of attributes. We deliberately implemented it that way so that existing code generation does not break with the introduction of template-type attributes.

For Java we already have an example how to do code generation for template type attributes in this PR: https://github.com/open-telemetry/semantic-conventions-java/pull/24